### DISCUSSION TITLE
Transfer issue in version 0.35

### DISCUSSION THREAD

**Discussion by Esmail0karimi** [horizontalsystems/unstoppable-wallet-android#6608] [General]
I had problem in last version 0.34.1 and I did can't transfer my issue and I had error scalar is not in the interval [1 , n-1]
And the customer support told me after update version 0.35 will solve your problem 
But unfortunately my problem still here and I have error scalar is not in the interval and I can't transfer or swap my asset
Can help me I need to transfer my asset 😑

**Comment by serrq** [comment]
#6606 Your problem has been labeled, so they will deal with it as soon as they have a way to get out of other jobs.
You need patience.

**Reply by Esmail0karimi** [reply]
When it will solved it's necessary for me I need to transfer my asset
